Lesson8_Dictionary/homework8_dictionary.py

```diff
@@ -0,0 +1,50 @@
+# Task 1
+print('Task 1: \'Roles and Users\'')
+roles = {
+    'admin': ['Olga', 'Julia'],
+    'maintainer': ['Oleg', 'Roman'],
+    'manager': ['Ruslan', 'Artem'],
+    'developer': ['Viktor', 'Alex']
+}
+
+name = input('Please to enter name with upper registr for first letter: ')
+role = []
+for key, value in roles.items():
+    if name in value:
+        role = key
+        break
+if role:
+    print(f'Hello, {role}')
+else:
+    print('Hello, Guest')
```

Comment:
Can be better
![image](https://github.com/annasever/python_cources/assets/51053483/f8b8690f-0359-493c-89ea-23a253a33c03)